src/pages/community/members.js

```diff
@@ -201,14 +201,27 @@ const options = [
 }));
 
 const activeMember = {
-  label: "Active",
+  label: (
+    <DropdownWrapper>
+      <div className="allOptions">Active</div>
+    </DropdownWrapper>
+  ),
   value: "active",
   color: theme.linkColor,
   isFixed: true,
   icon: activeIcon,
   className: "allOptions",
 };
 
+const allMembers = {
+  label: "All Members",
+  value: "all",
+  color: theme.linkColor,
+  isFixed: true,
+  icon: null,
+  className: "allOptions",
+};
```

Comment:
Why do we need this?

Reply:
👀 the current version of the filter when selected shows all members either active or inactive, but previously when the writer badge was not integrated it was only showing the active member. So I was testing with some logic used earlier if that can help in solving the issue. 

I mistakenly added this variable to the commit. If in today's meeting it is decided to show only active members then this part will be needed otherwise I will remove it.

Reply:
No need to wait till the meeting.
Remove the `allMember` object, the default value should be `activeMember`